cephadm-preflight.yml

```diff
@@ -19,20 +19,10 @@
 - hosts: all
   become: true
   gather_facts: true
-  vars:
-    ceph_origin: community
-    ceph_rhcs_version: 5
-    ceph_mirror: https://download.ceph.com
-    ceph_stable_key: https://download.ceph.com/keys/release.asc
-    ceph_release: pacific
-    upgrade_ceph_packages: false
-    ceph_pkgs:
-      - lvm2
-      - chrony
-      - cephadm
-      - podman
-      - python3
   tasks:
+    - import_role:
+        name: ceph-defaults
```

Comment:
Why is the `ceph-defaults` role not in a `roles` folder?

Reply:
could be in a roles folder or not, it doesn't really matter at the moment.
I don't mind moving it to a roles folder, in fact, perhaps it's better to keep the default behavior indeed